src/github/graphql.rs

```diff
@@ -0,0 +1,347 @@
+                            .map(|nodes| {
+                                nodes
+                                    .into_iter()
+                                    .filter_map(|node| node)
+                                    .map(|node| node.name)
+                                    .collect::<Vec<_>>()
+                            })
+                            .unwrap_or_default()
+                    })
+                    .unwrap_or_default();
+                if !labels.iter().any(|label| label == "T-compiler") {
+                    return None;
+                }
+                let labels = labels.join(", ");
+
+                let assignees: Vec<_> = pr
+                    .assignees
+                    .nodes
+                    .unwrap_or_default()
+                    .into_iter()
+                    .filter_map(|user| user)
+                    .map(|user| user.login)
+                    .collect();
+
+                let mut reviews = pr
+                    .reviews
+                    .map(|reviews| {
+                        reviews
+                            .nodes
+                            .map(|nodes| {
+                                nodes
+                                    .into_iter()
+                                    .filter_map(|n| n)
+                                    .map(|review| {
+                                        (
+                                            review
+                                                .author
+                                                .map(|a| a.login)
+                                                .unwrap_or("N/A".to_string()),
+                                            review.created_at,
+                                        )
+                                    })
+                                    .collect::<Vec<_>>()
+                            })
+                            .unwrap_or_default()
+                    })
+                    .unwrap_or_default();
+                reviews.sort_by_key(|r| r.1);
+
+                let comments = pr
+                    .comments
+                    .nodes
+                    .map(|nodes| {
+                        nodes
+                            .into_iter()
+                            .filter_map(|n| n)
+                            .map(|comment| {
+                                (
+                                    comment.author.map(|a| a.login).unwrap_or("N/A".to_string()),
+                                    comment.created_at,
+                                )
+                            })
+                            .collect::<Vec<_>>()
+                    })
+                    .unwrap_or_default();
+                let mut comments: Vec<_> = comments
+                    .into_iter()
+                    .filter(|comment| assignees.contains(&comment.0))
+                    .collect();
+                comments.sort_by_key(|c| c.1);
```

Comment:
@jackh726 I've noticed that the last comment now reported is from an assignee. Before it was the absolute latest comment (and not always correct because it included also bots).

In case [of this issue](https://github.com/rust-lang/rust/pull/84944#issuecomment-906810332) the result is a bit confusing: the latest comment reported in the agenda is "4 months ago" but Wesley added a comment about a week ago.

How should we interpret now the latest comment? "Latest comment from assignee" or "latest comment from a human"?

cc: @spastorino

Reply:
@apiraino yeah, I think "latest comment from assignee" makes the most sense; will require a bit of changes to the template and such

We should maybe quickly note (maybe discuss) in the meeting that updates like Wesley's should probably be a "review" or should change the status to either blocked or waiting-on-author.

Reply:
Maybe comments from assignees + members of the org? :P

Reply:
@spastorino that wouldn't really work given that bors and highfive are members, as well as triage.

Reply:
Well yeah you could filter those out.

Reply:
I think it's easier/more straightforward to stick with last comment by assignee. If the goal of this is to move the PR to a resolution (merge or close), then I think *reviews* are the better metric.

Sure, some newer PRs may start with a discussion through comments, but as PRs age, that discussion should resolve, or be a blocker. At the limit of "very old" for PRs, what kind of comments are we expecting? #84944 is an interesting case; it's an old issue that *was* waiting for review, but when it was discussed during the meeting, was decided it's now blocked on a test. However, when that test is added, it's probably *still* "very old"; this comment doesn't necessarily "refresh" that.

Reply:
I can see how the "last comment by assignee" make sense in the context of PRs and issues. However the initial intent of appending the most recent comment was to check the pulse of an MCP/FCP and see if it was alive or abandoned (after ~3 months of inactivity they will be closed).

Changing now how the last comment works would perhaps require a split:
- last comment from assignee for PRs/issues
- last comment from a human for MCPs/FCPs

makes sense?

Reply:
I think we don't need to move away from last comment (by anyone) for issues or MCPs/FCPs. The simplest way to think about these is that these don't usually get assigned *anyways*.

I'm really not seeing that there is anything that needs to be changed right now, other than just a rewording to the template to make it clear that it's not the last comment, but actually "last review activity" (even "last comment by assignee" is wrong, I'm remembering, since this takes into account reviews too)

Reply:
ok, if you're happy with how it works then I'm happy, too :) 

I'll push a small patch to update the wording of the template as you suggest.

thanks for answering quickly to my doubt!